### ISSUE TITLE
MAF>0.04% filter is too strict in some smaller ExAC subpopulations

### ISSUE THREAD

**Issue by ckandoth** [mskcc/vcf2maf#46]
What the title says

**Comment by ckandoth** [comment]
Don't have an example mutation where this actually caused an issue... But may find one thru PanCanAtlas

**Comment by ckandoth** [comment]
```text
# ExAC subpopulation counts and different fractions of them:
#Pop Samples 1%  5%   10%  20%  30%   40%   50%
AFR  5203    52  260  520  1041 1561  2081  2602
AMR  5789    58  289  579  1158 1737  2316  2895
EAS  4327    43  216  433  865  1298  1731  2164
FIN  3307    33  165  331  661  992   1323  1654
NFE  33370   334 1669 3337 6674 10011 13348 16685
SAS  8256    83  413  826  1651 2477  3302  4128
OTH  454     5   23   45   91   136   182   227
```

**Comment by ckandoth** [comment]
`FIN` and `OTH` are the 2 smallest subpopulations. In `FIN` with 3307 individuals (6614 alleles), a 0.04% AF cutoff requires 3 or more alleles as grounds for removal... which is fair. So only `OTH` is problematic. So as a fix, I just removed `OTH` as one of the subpops that are checked. The individuals in `OTH` will still contribute to the global AF, so it's no biggie. This fix will go into v1.6.7
